tests/actions/PolicyTest.ts

```diff
@@ -20,25 +21,27 @@ describe('actions/Policy', () => {
     });
 
     beforeEach(() => {
+        // @ts-expect-error TODO: Remove this once TestHelper (https://github.com/Expensify/App/issues/25318) is migrated to TypeScript.
```

Comment:
Don't put TODOs in the code. If we have an issue, that's already good enough

Reply:
@hayata-suenaga any reason we need TODOs in the code, or are GH issues sufficient? I ask cause we normally don't put TODOs in code (see [SO](https://stackoverflowteams.com/c/expensify/questions/14401)), and this guideline is kinda contradicts that. In any case, @HezekielT don't block on this, and sorry for the confusion

Reply:
thanks @blazejkustra for the explanation!

Reply:
Thanks for the context. I'm still curious, why do we need TODOs when we have GH issues? Don't the GH issues serve as TODOs?